app/serializers/api/participant_serializer.rb

```diff
@@ -25,7 +25,9 @@ class AttributesSerializer < Blueprinter::Base
           end
         end
 
-        field(:trn, name: :teacher_reference_number)
+        field(:teacher_reference_number) do |object, _options|
```

Comment:
It can be for a future ticket, but we should DRY up this serializer/spec so we're not repeating the same fields for each view, similar to how its done in the declaration serializer https://github.com/DFE-Digital/npq-registration/blob/6aaca99983a36898948db3d4ed264e7f78aa051a/app/serializers/api/declaration_serializer.rb

Reply:
will add to my notes 👍